DigitalLearningSolutions.Web/ViewModels/LearningMenu/InitialMenuViewModel.cs

```diff
@@ -39,5 +41,21 @@ public InitialMenuViewModel(CourseContent courseContent)
                 courseContent.TutorialsCompletionThreshold
             );
         }
+
+        private static string FormatDuration(int duration)
+        {
+            if (duration < 60)
+            {
+                return FormatTimeUnits(duration, "minute");
+            }
+
+            var formattedHours = FormatTimeUnits(duration / 60, "hour");
+            var formattedMinutes = FormatTimeUnits(duration % 60, "minute");
+            return duration % 60 == 0 ? formattedHours : formattedHours + formattedMinutes;
+
+        }
+
+        private static string FormatTimeUnits(int duration, string unit) =>
+            duration == 1 ? $" {duration} {unit}" : $" {duration} {unit}s";

```

Comment:
Any reason why you've included the whitespace here? Not very important but I found it a bit surprising, so would have probably left it to be defined in the cshtml as before. It makes the average duration make more sense on its own 🙂

Reply:
This was to make adding them together nice, but I agree it turned out to be a bit weird and I've changed it in 1341b0f 🙂